library/src/main/java/com/chuckerteam/chucker/internal/support/ClearDatabaseService.kt

```diff
@@ -1,15 +1,20 @@
 package com.chuckerteam.chucker.internal.support
 
-import android.app.IntentService
+import android.content.Context
 import android.content.Intent
+import androidx.core.app.JobIntentService
 import com.chuckerteam.chucker.internal.data.repository.RepositoryProvider
 import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.launch
 
-internal class ClearDatabaseService : IntentService(CLEAN_DATABASE_SERVICE_NAME) {
+internal class ClearDatabaseService : JobIntentService() {
     private val scope = MainScope()
 
-    override fun onHandleIntent(intent: Intent?) {
+    fun enqueueWork(context: Context, work: Intent) {
```

Comment:
I think this should be a method on the companion object. Otherwise it is awkward to use with instantiating the service in order to enqueue work on it.

Reply:
Agree